### PR TITLE
Recover Account Change Password

### PR DESCRIPTION
**Pull request by shreeyash07**
## Addresses:
- [Recover Account](https://github.com/orgs/toggle-corp/projects/8/views/23?pane=issue&itemId=39896979)

## Changes
- Add Recover Password page

## This PR doesn't introduce:
- [x] typos
- [x] conflict markers
- [x] unwanted comments
- [x] temporary files, auto-generated files or secret keys
- [x] `console.log` meant for debugging
- [x] codegen errors

### CODE REVIEW

File: src/views/RecoverPassword/index.tsx [hunk 1]
```diff
@@ -0,0 +1,184 @@
...
+            required: true,
+            requiredValidation: requiredStringCondition,
+        },
+        confirmPassword: {
+            required: true,
+            requiredValidation: requiredStringCondition,
+            validations: [samePasswordCondition],
+        },
+    }),
+};
```
Comment (tnagorra) [inline src/views/RecoverPassword/index.tsx]: This is not the correct way to show dependent validation error.

File: src/views/RecoverPassword/index.tsx [hunk 2]
```diff
@@ -0,0 +1,184 @@
...
+    fields: (): FormSchemaFields => ({
+        newPassword: {
+            required: true,
+            requiredValidation: requiredStringCondition,
+        },
+        confirmPassword: {
+            required: true,
+            requiredValidation: requiredStringCondition,
+            validations: [samePasswordCondition],
+        },
```
Comment (tnagorra) [inline src/views/RecoverPassword/index.tsx]: Also, we should force the confirmPassword field to be undefined.

Comment (tnagorra) [reply src/views/RecoverPassword/index.tsx]: Please refer to account create page.

**Review by tnagorra — CHANGES_REQUESTED** [review]

File: src/views/RecoverPassword/i18n.json [hunk 3]
```diff
@@ -0,0 +1,13 @@
+{
+    "namespace": "recoverPassword",
+    "strings": {
+        "pageTitle": "IFRC GO - Recover Password",
+        "pageHeading": "Recover Account",
+        "newPassword": "New Password",
+        "confrimPassword": "Confirm new Password",
+        "submitButtonLabel": "Submit",
+        "successfulMessageTitle": "Password Changed!",
+        "successfulMessageDescription": "Your password has been changed Please login again!",
```
Comment (barshathakuri) [inline src/views/RecoverPassword/i18n.json]:
```suggestion
        "successfulMessageDescription": "Your password has been changed. Please login again!",
```

**Pull request merged**